website/blog/2018-09-26-circleci-ci-cache-proxy.md

```diff
@@ -0,0 +1,53 @@
+---
+title:     "Dependencies caching on local CircleCI builds"
+author:    Manuel Mazzuola
+authorURL: https://keybase.io/originof
+---
+
+At Cloudesire a couple of years ago we decided to switch from Jenkins to CircleCI. First of all, because we are lazy to self-maintain a server and secondly because CircleCI seemed a great build service and they were very generous with their limits.
+
+A great tool CircleCI provides is the local CLI.  
+The `circleci` commands enable you to reproduce the CircleCI environment locally and run jobs as if they were running on the hosted application for more efficient debugging and configuration in the initial setup phase.  
+The bad news about the local CLI is that the `save_cache` and `restore_cache` keys are [ignored](https://circleci.com/docs/2.0/local-cli/#caching) in the local builds and dependencies are downloaded each time, shame 🕭 shame 🕭 shame 🕭.
+
+A fast setup solution I currently use is to proxy docker to a local nginx or squid container that acts as cache layer.
+
+### Start the proxy container
+
+`$ docker run --name nginx-cache -d manuelmazzuola/nginx-cache` and get the internal IP by inspecting it with `docker inspect nginx-cache | grep IPAddress`
```

Comment:
give these commands their own boxes with the new line / four leading spaces format